app/components/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,58 @@
+import { observer } from "mobx-react";
```

Comment:
A very general confirmation dialog. Good for simple use cases like interrupting a sensitive config change